fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/detail/event/election/adapters/CastVoteViewPagerAdapter.java

```diff
@@ -119,8 +120,9 @@ public int getItemCount() {
   private boolean checkEachQuestion() {
     List<Integer> allVotes = mLaoDetailViewModel.getCurrentElectionVotes().getValue();
     if (allVotes != null) {
+      if (allVotes.size() < getItemCount()) return false;
```

Comment:
why don't we just return false if allVotes.isEmpty() here  ? Instead of using a hard coded value -1 ?